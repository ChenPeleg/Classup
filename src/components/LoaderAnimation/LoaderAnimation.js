import React from "react";
import styles from "./LoaderAnimation.module.scss";

const LoaderAnimation = (props) => {
  return (
    <div>
      <br></br>
      <br></br>
      <br></br>
      <div className={styles.loader}></div>Loading...
    </div>
  );
};
export default LoaderAnimation;
