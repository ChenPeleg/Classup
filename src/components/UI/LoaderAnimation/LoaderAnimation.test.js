import React from "react";
import { shallow } from "enzyme";
import LoaderAnimation from "./LoaderAnimation";

const loader = shallow(<LoaderAnimation />);
describe("<LoaderAnimation>", () => {
  it("renders correctly", () => {
    expect(loader).toMatchSnapshot();
  });
});
