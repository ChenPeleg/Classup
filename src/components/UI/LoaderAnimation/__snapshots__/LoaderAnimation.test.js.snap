// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoaderAnimation> renders correctly 1`] = `
<div>
  <br />
  <br />
  <br />
  <div
    className="loader"
  />
  Loading...
</div>
`;
